ownership-chain/runtime/src/lib.rs

```diff
@@ -144,19 +143,14 @@ pub type UncheckedExtrinsic =
 pub type CheckedExtrinsic =
 	fp_self_contained::CheckedExtrinsic<AccountId, RuntimeCall, SignedExtra, H160>;
 
-type Migrations = (
-	pallet_collator_selection::migration::v1::MigrateToV1<Runtime>,
-	migrations::v1::version_unchecked::MigrateSudo<Runtime>,
-);
-
 /// Executive: handles dispatch to the various modules.
 pub type Executive = frame_executive::Executive<
 	Runtime,
 	Block,
 	frame_system::ChainContext<Runtime>,
 	Runtime,
 	AllPalletsWithSystem,
-	Migrations,
+	pallet_collator_selection::migration::v1::MigrateToV1<Runtime>,
```

Comment:
we still need this migration, and it doesn't harm us doing it

Reply:
actually, I don't think it runs it at all if we spawn the new chain, so removing it